http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -231,6 +231,15 @@ fi
 
 mount -t ext4 -O discard,defaults /dev/${DISK_DEVICE_ID} ${WORK_DIRECTORY}
 
+# Resize persistent disk if needed.
```

Comment:
Adding some logic to that PD integration test which writes a file, then reads it back would be great as well.